assets/css/app.css

```diff
@@ -1862,20 +1874,6 @@ footer .refs__links__link__logo {
   font-size: 1.75rem;
 }
 
-.refs__button {
```

Comment:
Tohle to samé, házelo se dolu kvůli zachování barvy tlačítka, proč se teď stěhuje zase nahoru?